components/article/client/image_set.coffee

```diff
@@ -12,7 +15,10 @@ module.exports = class ImageSetView extends Backbone.View
     { @items, @user } = options
     @length = @items.length
     @currentIndex = 0
+    @followArtists = new FollowArtists []
     @render()
+    @setupFollowButtons()
+    console.log('setupFollowButtons')
```

Comment:
Will remove that in a minute, thanks!

Sent from my iPhone

> On Sep 9, 2016, at 5:44 PM, Charles Broskoski <notifications@github.com> wrote:
> 
> In components/article/client/image_set.coffee:
> 
> >      @render()
> > +    @setupFollowButtons()
> > +    console.log('setupFollowButtons')
> 🌴
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>